src/test/java/fi/otavanopisto/kuntaapi/test/AbstractMocker.java

```diff
@@ -0,0 +1,280 @@
+   * @param content response content
+   */
+  public void mockGetString(String path, String type, String content) {
+    stringMocks.add(new StringGetMock(path, type, content, null));
+  }
+  
+  /**
+   * Mocks string response for GET request on path
+   * 
+   * @param path path
+   * @param type response content type 
+   * @param queryParams query params for the reuqest
+   * @param content response content
+   */
+  public void mockGetString(String path, String type, String content, Map<String, String> queryParams) {
+    stringMocks.add(new StringGetMock(path, type, content, queryParams));
+  }
+  
+  /**
+   * Mocks JSON response for GET request on path
+   * 
+   * @param path path
+   * @param object JSON object
+   */
+  public void mockGetJSON(String path, Object object, Map<String, String> queryParams) {
+    try {
+      stringMocks.add(new StringGetMock(path, "application/json", new ObjectMapper().writeValueAsString(object), queryParams));
+    } catch (JsonProcessingException e) {
+      logger.log(Level.SEVERE, "Failed to serialize mock JSON object", e);
+      fail(e.getMessage());
+    }
+  }
+  
+  /**
+   * Reads JSON file as organization object
+   * 
+   * @param file path to JSON file
+   */    
+  public Organization readOrganizationFromJSONFile(String file) {
+    return readJSONFile(file, Organization.class);
+  }
+  
+  /**
+   * Reads JSON file as organization object
+   * 
+   * @param file path to JSON file
+   */    
+  public Service readServiceFromJSONFile(String file) {
+    return readJSONFile(file, Service.class);
+  }
+  
+  /**
+   * Reads JSON file as electronic service channel object
+   * 
+   * @param file path to JSON file
+   */    
+  public ElectronicChannel readElectronicChannelFromJSONFile(String file) {
+    return readJSONFile(file, ElectronicChannel.class);
+  }
+  
+  /**
+   * Reads JSON file as phone service channel object
+   * 
+   * @param file path to JSON file
+   */    
+  public PhoneChannel readPhoneChannelFromJSONFile(String file) {
+    return readJSONFile(file, PhoneChannel.class);
+  }
+
+  /**
+   * Reads JSON file as printable form service channel object
+   * 
+   * @param file path to JSON file
+   */    
+  public PrintableFormChannel readPrintableFormChannelFromJSONFile(String file) {
+    return readJSONFile(file, PrintableFormChannel.class);
+  }
+
+  /**
+   * Reads JSON file as printable form service channel object
+   * 
+   * @param file path to JSON file
+   */    
+  public ServiceLocationChannel readServiceLocationChannelFromJSONFile(String file) {
+    return readJSONFile(file, ServiceLocationChannel.class);
+  }
+
+  /**
+   * Reads JSON file as printable form service channel object
+   * 
+   * @param file path to JSON file
+   */    
+  public WebPageChannel readWebPageChannelFromJSONFile(String file) {
+    return readJSONFile(file, WebPageChannel.class);
+  }
+  
+  private <T> T readJSONFile(String file, Class <T> type){
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "readJSONFile" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)